.gitignore

```diff
@@ -1,11 +1,9 @@
-.history
+venv/
 build/
 *.charm
-charms/metallb-controller/.build/
-charms/metallb-speaker/.build/
-.tox
-.coverage*
-.stestr/
-__pycache__
-coverage
-cover
+.tox/
+.coverage
+__pycache__/
+*.py[cod]
+.idea
+.vscode/
```

Comment:
i kinda fall into this camp: https://blog.martinhujer.cz/dont-put-idea-vscode-directories-to-projects-gitignore/

You have your own machine that's got programs on it -- the future of the project shouldn't guess what your editor is

```suggestion
```